general/open-source.md

```diff
@@ -6,11 +6,25 @@ Open source software and open standards are critical components of modern web ap
 We believe relying upon open standards and software has been fundamental to the success of the software industry as a whole. To aid in this, Langley Foxall is committed to giving back to the community. We try to contribute useful open source software, libraries, and other tools wherever possible. These packages are often used internally or as a part of our bespoke software solutions.
 
 In fact, the page you are reading right now is part of just one of these open source projects.
+
 ## Our Open Source Projects
 
 We love contributions and want our packages to help people, so if you have suggestions for improvements, please file
 an issue on the relevant repository.
 
+## Creating Open Source Projects
+
+When creating open source software, we're aware that these projects can be used in systems that other developers build & rely on, because of this, as of 26/01/2018 we have rigid requirements for our oss which prevent breaking changes from being introduced into stable versions of the public projects we release.
+
+When a project reaches `v1.0.0` we have the following safe guards in place:
+
+- When a project reaches `v1.0.0`, we stricly follow semantic versioning to ensure updates don't break existing implementations or expected functionality.
```

Comment:
Whoops, missed this one. 

> When a project reaches `v1.0.0`

Why is this repeated?

Reply:
Because before a project reaches v1.0.0, we don't follow semantic versioning as the project has not been released and we simply work with master / feature branches, you only apply semantic versioning to release.

This could be reworked to make it a bit clearer.

Reply:
Due to the following section of the SemVer documentation, we can state we always use sematic versioning rather than just at >= v1.0.0.

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

Reply:
🎉